pom.xml

```diff
@@ -1345,6 +1345,211 @@
+                  <old>method void io.netty.channel.epoll.EpollEventLoop::openFileDescriptors()</old>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.finalMethodAddedToNonFinalClass</code>
+                  <new>method io.netty.util.concurrent.Future&lt;java.lang.Void&gt; io.netty.channel.SingleThreadIoHandleEventLoop::registerForIo(io.netty.channel.IoHandle) @ io.netty.channel.epoll.EpollEventLoop</new>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.nowFinal</code>
+                  <old>method void io.netty.channel.epoll.EpollEventLoop::wakeup(boolean)</old>
+                  <new>method void io.netty.channel.SingleThreadIoHandleEventLoop::wakeup(boolean) @ io.netty.channel.epoll.EpollEventLoop</new>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>ava.class.nonFinalClassInheritsFromNewClass</code>
+                  <old>class io.netty.channel.epoll.EpollEventLoop</old>
+                  <new>class io.netty.channel.epoll.EpollEventLoop</new>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.returnTypeChangedCovariantly</code>
+                  <old>method io.netty.channel.EventLoop io.netty.channel.MultithreadEventLoopGroup::next() @ io.netty.channel.epoll.EpollEventLoopGroup</old>
+                  <new>method io.netty.channel.IoHandleEventLoop io.netty.channel.MultiThreadIoHandleEventLoopGroup::next() @ io.netty.channel.epoll.EpollEventLoopGroup</new>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.class.nonFinalClassInheritsFromNewClass</code>
+                  <old>class io.netty.channel.epoll.EpollEventLoop</old>
+                  <new>class io.netty.channel.epoll.EpollEventLoop</new>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.finalMethodAddedToNonFinalClass</code>
+                  <new>method io.netty.channel.epoll.EpollRegistration io.netty.channel.epoll.AbstractEpollChannel::registration() @ io.netty.channel.epoll.AbstractEpollServerChannel</new>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.finalMethodAddedToNonFinalClass</code>
+                  <new>method io.netty.channel.epoll.EpollRegistration io.netty.channel.epoll.AbstractEpollChannel::registration() @ io.netty.channel.epoll.AbstractEpollStreamChannel</new>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.class.nonPublicPartOfAPI</code>
+                  <new>interface io.netty.channel.epoll.EpollRegistration</new>
+                </item>
+
```

Comment:
this is huge, epic.

Reply:
While the list to ignore is very big I think in real world it will not affect people